<?xml version="1.0"?>
<launch>
<arg name="gemweightspath" value="/home/yujc/robotws/DSLAM_release/src/ROS-DSLAM/script/ROS_GEM/weights/gem.pth" />
<arg name="datasetpath" value="/home/yujc/robotws/dataset/image_503_loop/" />

  <node name="publish" pkg="dslam_sp" required="true" type="publish_merge" args="-p $(arg datasetpath)"/>
  
  <node name="sp_ORB" pkg="dslam_sp" required="true" type="orb_EF" output="screen">
    <remap from="image" to="/merge/img_depth_file" />
  </node>

  
  <node name="vo" pkg="dslam_sp" required="true" type="visual_odometry_orb" output="screen">
    <remap from="feature" to="/orb_EF/featurepoints_descriptors" />
  </node>

  <node name="gen_tf" pkg="dslam_sp" required="true" type="generate_track_node.py" output="screen">
    <remap from="relpose" to="/visual_odometry/transform_relative" />
  </node>


</launch>
